app/controllers/api/project.php

```diff
@@ -144,6 +145,22 @@
             ];
         }, $dbForProject->find('buckets'));
 
+        $deploymentsStorageBreakdown = array_map(function ($function) use ($dbForProject) {
```

Comment:
not sure we need to call it `deploymentsStorageBreakdown` because `bucketsBreakdown` already captures the fact that its a breakdown of the buckets storage... So you might as well just use `deploymentsBreakdown`

Reply:
Not sure I agree, we already have `deployments`, `deploymentsTotal`, etc... etc... I don't think it'll be clear in the code that this is for the storage specifically, in the buckets we know we're only tracking storage but in the deployments we're not sure if we're tracking storage or how many exist